src/ending.c

```diff
@@ -65,59 +65,63 @@ void bad_ending_marisa(Ending *e) {
 
 	add_ending_entry(e, -1, "Marisa closed the book and let it slam onto the floor.", NULL);
 	add_ending_entry(e, -1, "Patchouli: “Mind your manners! That’s a very old tome!”", NULL);
-	add_ending_entry(e, -1, "Marisa: “It’s so irritatin’! I thought I solved everything but now that tower’s just sittin’ there, ’n the whole Mountain’s been quarantined…”", NULL);
-
+	add_ending_entry(e, -1, "Marisa: “It’s so irritatin’! I thought I solved everything but now that tower’s just sittin’ there, ‘n the whole Mountain’s been quarantined…!”", NULL);
 	add_ending_entry(e, -1, "Marisa: “It just ain’t satisfyin’, y’know?”", NULL);
-	add_ending_entry(e, -1, "Flandre: “I could destroy it! Make it go BOOM!”", NULL);
-	add_ending_entry(e, -1, "Marisa: “Gah! Now what did I say about sneakin’ up on people, kiddo?”", NULL);
-	add_ending_entry(e, -1, "Flandre: “Sorry!”", NULL);
+
+	add_ending_entry(e, -1, "Flandre: “I could destroy it, you know.”", NULL);
+	add_ending_entry(e, -1, "Marisa: “Gah! Now what did I say about sneakin’ up on your friends, Flan?”", NULL);
+	add_ending_entry(e, -1, "Flandre: “Sorry, sorry!”", NULL);
```

Comment:
I think I actually prefer the previous version of this; it has more character, I think? The "Make it go BOOM!" line in particular feels very "Flandre" to me, I can easily picture her saying that with exaggerated childish expressiveness. It's endearing.

Do you think there's anything particularly off about this bit that you decided to change it?

Reply:
My main worry was that I'd made her a little TOO childish? It mostly stems from me re-reading her EoSD dialogue and some of the new manga and getting self-conscious of my characterization.

But if you liked it, I can easily put it back. I thought it was cute too.

Reply:
Don't be too hard on yourself about characterization. I think it's not really important to keep it super consistent with the canon. I'd say the canon itself isn't always self-consistent, not to mention the countless fanworks.

I see Touhou fanworks as a kind of modern folklore. Being able to reinterpret characters in different ways is a beauty of it. A slightly more childish Flandre hardly compares to the liberties we took with the PC-98 folk, anyway :)

Reply:
Reverted those changes!